spec/registrant_spec.rb

```diff
@@ -0,0 +1,142 @@
+require './lib/registrant'
+require './lib/facility'
+
+RSpec.describe Registrant do 
+
+  it 'exists' do 
+    registrant_1 = Registrant.new('Bruce', 18, true )
+
+    expect(registrant_1).to be_a(Registrant)
+  end
+
+  it 'has readable attributes' do 
+    registrant_1 = Registrant.new("Bruce", 18, true )
+    
+
+    expect(registrant_1.name).to eq("Bruce")
+    expect(registrant_1.age).to eq(18)
+    expect(registrant_1.permit?).to eq(true)
+    expect(registrant_1.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+  
+  end 
+    
+  it '#earn_permit' do 
+
+    registrant_2 = Registrant.new('Penny', 15 )
+    expect(registrant_2.name).to eq("Penny")
+    expect(registrant_2.age).to eq(15)
+    expect(registrant_2.permit?).to eq(false)
+    expect(registrant_2.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    
+    registrant_2.earn_permit 
+    expect(registrant_2.permit?).to eq(true)
+  end
+
+  it 'can issue written test' do 
```

Comment:
These are the tests that would live in your `facility_spec` because the methods are written in the `Facility` class.  So we would need to Unit test them there.